CoreFoundation/Base.subproj/CFRuntime.h

```diff
@@ -195,10 +195,18 @@ typedef struct __CFRuntimeBase {
     uintptr_t _cfisa;
     uintptr_t _swift_rc;
     // This is for CF's use, and must match _NSCFType layout
+#if __LP64__
```

Comment:
Presumably `LLP64`, too.

Reply:
Yes, for Windows also, maybe we must add an || condition.

Reply:
I agree

Reply:
Looks like the .c file was updated but not the .h file?

Reply:
Is that a question to me?